tests/suite-integration/Entity.php

```diff
@@ -31,7 +31,7 @@
 
 namespace tests\units;
 
-use Glpi\Test\CommonTestCase;
+use Flyvemdm\Test\CommonTestCase;
```

Comment:
Flyvemdm\Test\CommonTestCase inherits from Glpi\Test\CommonTestCase and will allow us to add some helper methods for unit tests